aspnet/client-side/bundling-and-minification.rst

```diff
@@ -1,10 +1,177 @@
+``Include("~/Scripts/Common/*og.*")``                Invalid pattern exception. Only one wildcard character is allowed. 
+``Include("~/Scripts/Common/T*")``                   *ToggleDiv.js*, *ToggleImg.js* 
+``Include("~/Scripts/Common/*")``                    Invalid pattern exception. A pure wildcard segment is not valid. 
+``IncludeDirectory("~/Scripts/Common", "T*")``       *ToggleDiv.js*, *ToggleImg.js* 
+``IncludeDirectory("~/Scripts/Common", "T*",true)``  *ToggleDiv.js*, *ToggleImg.js*, *ToggleLinks.js* 
+===================================================  ===========================================================================================  
+
+Explicitly adding each file to a bundle  is generally the preferred over wildcard loading of files for the following reasons:
+
+- Adding scripts by wildcard defaults to loading them in alphabetical order, which is typically not what you want. CSS and JavaScript files frequently need to be added in a specific (non-alphabetic) order. You can mitigate this risk by adding a custom  IBundleOrderer implementation, but explicitly adding each file is less error prone. For example, you might add new assets to a folder in the future which might require you to modify your  IBundleOrderer implementation.
+- View specific files added to a directory using wild card loading can be included in all views referencing that bundle. If the view specific script is added to a bundle, you may get a JavaScript error on other views that reference the bundle. 
+- CSS files that import other files result in the imported files loaded twice. 
+
+Bundle Considerations
+---------------------
+
+Bundling and minification primarily improve the first page request load time. Once a webpage has been requested, the browser caches the assets (JavaScript, CSS and images) so bundling and minification won’t provide any performance boost when requesting the same page, or pages on the same site requesting the same assets. If you don’t set the expires header correctly on your assets, and you don’t use bundling and minification, the browsers freshness heuristics will mark the assets stale after a few days and the browser will require a validation request for each asset. In this case, bundling and minification provide a performance increase after the first page request. For details, see the blog `Using CDNs and Expires to Improve Web Site Performance <http://blogs.msdn.com/b/rickandy/archive/2011/05/21/using-cdns-to-improve-web-site-performance.aspx>`_.
+
+The browser limitation of six simultaneous connections per each hostname can be mitigated by using a  CDN. Because the CDN will have a different hostname than your hosting site, asset requests from the CDN will not count against the six simultaneous connections limit to your hosting environment. A CDN can also provide common package caching and edge caching advantages.
```

Comment:
That's old, I think browers do more than 6 now. Use the link in my original article to look up the current limit.